samples/link-unfurling-stt/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<Microsoft-App-Id>>",
+  "packageName": "com.microsoft.teams.linkunfurlingstt",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://example.azurewebsites.net",
+    "privacyUrl": "https://example.azurewebsites.net/privacy",
+    "termsOfUseUrl": "https://example.azurewebsites.net/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Link unfurling STT"
+  },
+  "description": {
+    "short": "Sample to test link unfurling in STT",
+    "full": "This sample demos the feature of link unfurling in share to teams"
+  },
+  "accentColor": "#FFFFFF",
+  "staticTabs": [
+    {
+      "contentUrl": "https://<<base-url>>/tab",
```

Comment:
We are already providing https in contentURL , should we change <<base-url> to <<domain-name>> to keep it consistent?